website/versioned_docs/version-0.79/view-managers.md

```diff
@@ -478,7 +478,7 @@ void CustomUserControlViewManager::ReactContext(IReactContext reactContext) noex
 ### More extensibility points
 
 - In some scenarios, a view manager might need to have more context at view creation time in order to decide what kind of control to instantiate.
-This can be achieved by having the view manager implement the [`IViewManagerCreateWithProperties`](IViewManagerCreateWithProperties) interface:
+This can be achieved by having the view manager implement the `IViewManagerCreateWithProperties` interface:
```

Comment:
Why did you remove this link? The file exists.

Reply:
I pushed commits which fixed this, there's a section in `.unbroken_exclusions` because of the links on this particular page.